new.html

```diff
@@ -193,18 +180,16 @@ <h3>About Powergrid</h3>
 	<script src="modal.js"></script>
 	<script src="alerts.js"></script>
 	<script src="tabs.js"></script>
+	<script src="gridEditor.js"></script>
 	<script>
-
-		var grid = Object.assign({
-			html: '',
-			dispatchEvent: function (event) {
-				if (this.el && typeof this.el.dispatchEvent == 'function') {
-					this.el.dispatchEvent(event);
-				}
-			},
-			init: function (event) {
-				console.log('init')
-				this.el = event.currentTarget;
+		// Application and state management
```

Comment:
Decided we can keep our app and state management global so it would be accessible from all the components.
I also using `window` to send events.